package.json

```diff
@@ -67,9 +67,7 @@
     "ws": "^0.8.0",
     "memory-streams": "^0.1.0",
     "dev-null": "^0.1.1",
-    "express": "^4.9.0"
-  },
-  "devDependencies": {
+    "express": "^4.9.0",
```

Comment:
With shrinkwrap, are we not allowed to have dev dependencies any longer?  Not that it's a huge deal, but it appears that the dev dependencies are mainly there for deploying the worker rather than running it.